### PR TITLE
us2-vehicles

### PR DESCRIPTION
**Pull request by YannRaulet**
styling pages

### CODE REVIEW

**Review by erguculu — DISMISSED** [review]

File: src/Repository/VehicleRepository.php [hunk 1]
```diff
@@ -19,22 +19,30 @@ public function __construct(ManagerRegistry $registry)
...
+    * @return Vehicle[] Returns an array of Vehicle objects
+    */
+    public function findByParticularVehicle(): array
     {
         return $this->createQueryBuilder('v')
-            ->andWhere('v.exampleField = :val')
-            ->setParameter('val', $value)
-            ->orderBy('v.id', 'ASC')
-            ->setMaxResults(10)
+            ->andWhere('v.type = :ParticularVehicle')
```
Comment (RomainClair) [inline src/Repository/VehicleRepository.php]: What's the point of having a parameter when in fact you don't have parameter ?
And if you want vehicule by type, you don't need specific queries findBy should be enough

**Review by RomainClair — COMMENTED** [review]
Don't you have some unused code (custom queries) in your VehiculeRepository ?

File: src/Repository/VehicleRepository.php (hunk 1 above)
Comment (YannRaulet) [reply src/Repository/VehicleRepository.php]: an oversight on my part, it was a test.
I forgot to remove this part

**Review by erguculu — DISMISSED** [review]

File: src/Controller/ProductController.php [hunk 2]
```diff
@@ -5,6 +5,10 @@
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
+use App\Repository\VehicleRepository;
+use App\Entity\Vehicle;
+use App\Repository\chargingStations;
```
Comment (RomainClair) [inline src/Controller/ProductController.php]: Should Probably be ChargingStationsRepository (or not be there)

**Review by RomainClair — CHANGES_REQUESTED** [review]

File: src/Controller/ProductController.php (hunk 2 above)
Comment (YannRaulet) [reply src/Controller/ProductController.php]: I don't really need it actually

**Review by erguculu — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**